modules/testcontainers/src/main/resources/genesis.json

```diff
@@ -33,7 +32,7 @@
                       }
                     },
                     "signatories": [
-                      "ed01207233bfc89dcbd68c19fde6ce6158225298ec1131b6a130d1aeb454c1ab5183c0"
+                      "ed0120ce31629c9f44aac7bd3c043f0eca904b016ac95306ae285828f6f8903f7aa9ad"
```

Comment:
why this changed?

Reply:
It has been copied from iroha2 repository, but it doesn't affect anything